pkg/controller/info/info.go

```diff
@@ -50,12 +50,25 @@ func maskUser(s, username string) string {
 	return strings.ReplaceAll(s, username, "(USER)")
 }
 
-func (c *Controller) Info(_ context.Context, _ *logrus.Entry, param *config.Param) error { //nolint:funlen
+func getCurrentUserName() (string, error) {
 	currentUser, err := user.Current()
 	if err != nil {
-		return fmt.Errorf("get a current user: %w", err)
+		return "", fmt.Errorf("get a current user: %w", err)
 	}
 	userName := currentUser.Username
+	if runtime.GOOS == "windows" {
+		// On Windows, the user name is in the form of "domain\user".
+		const domainUserPartCount = 2
+		userName = strings.SplitN(userName, "\\", domainUserPartCount)[domainUserPartCount-1]
```

Comment:
I think strings.Cut is better than strings.SplitN.

https://pkg.go.dev/strings#Cut

```suggestion
		domain, un, ok := strings.Cut(userName, "\\")
		if ok {
			userName = un
		} else {
			userName = domain
		}
```

Reply:
Ah, I forgot that function exists.